content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -298,48 +298,23 @@ Configure the Journals
 Configure the Connection
 ************************
 
-First, you must select which users can access the **CodaBox** connection settings. To do so, go to
-:menuselection:`Settings --> Users & Companies --> Groups`, and search for the group
-:guilabel:`Allow to show and modify the CodaBox Connection Settings`. Add the users you want
-to this group. By default, only the :guilabel:`Administrator` is part of this group.
+#. Go to :menuselection:`Accounting --> Configuration --> Accounting: CodaBox`.
+#. Click on :guilabel:`Manage Connection` to open the connection wizard which shows the
+   :guilabel:`Accounting Firm VAT` number and the :guilabel:`Company VAT` number that will be used
+   for the connection.
+#. If this is your **first connection**, you can simply click on the :guilabel:`Create connection`.
+   The wizard confirms that the connection has been created on **Odoo's side**. Follow the steps
+   to validate the connection on **CodaBox's side** too.
 
-Then, to configure the connection itself, go to
-:menuselection:`Accounting --> Configuration --> Accounting: CodaBox`.
+   If this is **not your first connection**, the :guilabel:`Accounting Firm Password` provided by
+   Odoo during the first connection will be requested to create a new connection.
 
-The procedure differs depending on whether **CodaBox** is
-being setup for a first client or an additional one.
-
-.. tabs::
-
-   .. tab:: First connection
-
-       #. Click on :guilabel:`Connect`, where you are redirected to
-          the **CodaBox** platform, here validate the connection.
-       #. Once redirected back to Odoo, the :guilabel:`Status` should be set to
-          :guilabel:`Connected` and a new field :guilabel:`Access Token` should appear.
-
-       .. image:: belgium/codabox_configuration_connection_ok.png
-          :align: center
-          :alt: Configured connection.
-
-       .. note::
-          This :guilabel:`Access Token` will be used to create new connections
-          for other clients without having to validate the connection again.
-
-   .. tab:: Following connection
-
-       #. Copy the :guilabel:`Access Token` from a previous connection.
-       #. Click on :guilabel:`Connect`.
+      .. note::
+        This :guilabel:`Accounting Firm Password` is unique to Odoo and must be stored securely
+        on your side.
 
-      The **CodaBox** connection linked to the :guilabel:`Access Token` should still exist,
-      therefore there is no need to validate the connection again. A new and different
-      :guilabel:`Access Token` is generated, and the :guilabel:`Status` should be set to
-      :guilabel:`Connected`.
 
-       .. note::
-          By having a different :guilabel:`Access Token` for each client, you can
-          easily revoke the access of a client to your CodaBox account without
-          affecting the other clients. One client cannot interfere with another.
+#. The :guilabel:`Status` should have now switched to :guilabel:`Connected`.
```

Comment:
Since this is not an actual step to be performed by the user, I would take it out of the list.